pointers/pointers.c

```diff
@@ -66,7 +132,30 @@ int string_compare(char *m, char *n)
 */
 char *find_string(char *haystack, char *needle)
 {
-
+    int j = 0;
+    int count = 0;
+    char *ptr = NULL;
+    for (int i = 0; i < strlen(haystack); i++)
+    {
+        if (haystack[i] == needle[j])
+        {
+            if (j == 0)
+            {
+                ptr = &haystack[i];
+            }
+
+            j++;
+            count++;
+            printf("we are returning this: %d \n", strlen(needle));
+            printf("we are returning this: %d \n", count);
+            if (count == strlen(needle))
+            {
+                printf("we are returning this: %s \n", ptr);
+                return ptr;
+            }
+        }
+    }
+    return NULL;
 }
 
```

Comment:
Great job on competing pointers! I'm glad that you were able to get this figured out yesterday